README.md

```diff
@@ -53,6 +53,38 @@ You can first localize the robot by initializing the pose and then setting a nav
 
 [![](media/localize.png)](media/localize.mp4)
 
+## Running the reconnaissance demos:
+
+### Host/container insider attacker
+``` bash
+rocker rosswg/turtlebot3_demo "byobu -f configs/unsecure.conf attach"
```

Comment:
Does this container need not be connected to the same overlay network?

Reply:
No, this one aims to demontrate the attack from inside of the robot (inside of the container).